service/src/main/kotlin/fi/espoo/evaka/occupancy/Occupancy.kt

```diff
@@ -384,19 +384,30 @@ private fun <K : OccupancyGroupingKey> Database.Read.calculateDailyOccupancies(
 
     val serviceNeeds = this.createQuery(
         """
-SELECT sn.placement_id, sno.occupancy_coefficient, daterange(sn.start_date, sn.end_date, '[]') AS period
+SELECT
+    sn.placement_id,
+    CASE
+        WHEN u.type && array['FAMILY', 'GROUP_FAMILY']::care_types[] THEN 1.75
+        WHEN extract(YEARS FROM age(ch.date_of_birth)) < 3 THEN coalesce(sno.occupancy_coefficient_under_3y, default_sno.occupancy_coefficient_under_3y)
```

Comment:
The join of `sno` is an inner join and `occupancy_coefficient_under_3y` is `NOT NULL` -> `coalesce` and joining `default_sno` is not needed